.github/workflows/nightly-tests.yml

```diff
@@ -13,4 +23,49 @@ jobs:
   run-simulations:
     uses: cosmos/gaia/.github/workflows/sims.yml@main
 
-    
+  run-vulncheck:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - uses: actions/setup-go@v4
+        with:
+          go-version: 1.19.1      
```

Comment:
// comment
Govulncheck does not work on go 1.18.x and does not target a specific go version.
We should run govulncheck with the same go version as we intend to build Gaia.

Reply:
Gaia on main is built with go 1.20 https://github.com/cosmos/gaia/blob/c26737455aa306a6e1c285b8860fc26898dcb1b4/Makefile#L24

Reply:
```suggestion
          go-version: 1.20.x
```

Reply:
Updated task to run against go v.1.20.3